test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -249,6 +262,21 @@ Describe "New-Item with links" -Tags @('CI', 'RequireAdminOnWindows') {
         Test-Path $FullyQualifiedLink | Should -BeFalse
     }
 
+    It "Should create symbolic link with name contains special char" {
+        New-Item -Path $tmpDirectory -Name $testlinkSrcSpName -ItemType File > $null
+        $FullyQualifiedLSrcSp | Should -Exist
+
+        New-Item -Path $FullyQualifiedLinkSp -Target $FullyQualifiedLSrcSp -ItemType SymbolicLink > $null
+        $FullyQualifiedLinkSp | Should -Exist
+
+        $expectedTarget = Join-Path -Path $tmpDirectory -ChildPath $testlinkSrcSpName
+
+        $fileInfo = Get-ChildItem -Path $FullyQualifiedLinkSp
+        $fileInfo.Target | Should -Match ([regex]::Escape($expectedTarget))
```

Comment:
Why can't we do an -BeExactly assertion here instead of a -Match?

Reply:
I guess we can.
Actually this was created using the tests in the same file as reference:
https://github.com/PowerShell/PowerShell/blob/2737e74d864de909c7a359fad31e75e2cf0db34b/test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1#L217
https://github.com/PowerShell/PowerShell/blob/2737e74d864de909c7a359fad31e75e2cf0db34b/test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1#L242
Perhaps we have to review the New-Item test sometime.